draft-kuhn-quic-bdpframe-extension-01.xml

```diff
@@ -405,21 +424,17 @@ BDP Frame {
 
 		<t>
                 The server SHOULD NOT trust
-                the content of the BDP Frame received from the client. 
+                the content of the BDP Frame received from the client if the secured hash is not computed by the sender. 
```

Comment:
/not computed/not correctly recomputed? or something else?

Reply:
Fixed ! Thanks. https://github.com/NicoKos/resume_connection/commit/b6cf8df1573405b060e3cfcb62b1e0efcb38146d